srcpkgs/libupnpp/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'libupnpp'
 pkgname=libupnpp
-version=0.17.2
+version=0.18.0
 revision=1
 build_style=gnu-configure
 hostmakedepends="pkg-config"
-makedepends="expat-devel libcurl-devel libupnp-devel"
-short_desc="C++ wrapper for libupnp"
+makedepends="expat-devel libcurl-devel libnpupnp-devel"
+short_desc="C++ wrapper for libnpupnp"
 maintainer="amak <amak.git@outlook.com>"
 license="LGPL-2.1-or-later"
 homepage="https://www.lesbonscomptes.com/upmpdcli"
 distfiles="https://www.lesbonscomptes.com/upmpdcli/downloads/libupnpp-${version}.tar.gz"
-checksum=5abaaf353a1e9c3482d61ef2627b650285d59f27c1ee60d35b8951952261374f
+checksum=c9659cd36ce70c43e9889a6c3a5eeb28e4f799580727826a6c7aef9bef6b6937
```

Comment:
You do touch this, but it's an entirely separate concept. If your git-fu is adequate, please move the work on this into a new PR.